recipes-appfw/sample-node0/files/manifest.in

```diff
@@ -0,0 +1,21 @@
+{
+    "application": "node0",
+    "description": "node0 - A NodeJS sample app.",
+    "provider": "test",
+    "groups": "test-node0",
+    "environment": {
+        "NODE0_FOO": "node0-foo",
+        "NODE0_BAR": "node0-bar",
+    },
+    "command": [
```

Comment:
Sorry, my terminology was inconsistent/vague. By full containers I meant a system-container which is not running a dedicated image of its own (like you mentioned above) but instead running a 'filesystem view' that was stitched together (typically by systemd-nspawn) using a set of overlay- and bind-mounts on top of the basic host OS image.